sunpy/net/helio/hec.py

```diff
@@ -84,9 +84,9 @@ def __init__(self, link=None):
         if link is None:
             # The default wsdl file
             link = parser.wsdl_retriever()
-        # Disable SSL check.
         session = Session()
-        session.verify = True
+        # Disable SSL check.
+        session.verify = False
```

Comment:
OK, this has to stop. We can't keep doing this when their servers fall over.

We should make this configurable somehow so we can set it for the test run or something.

Reply:
Hmm, could make it an env var we set in tox?

Reply:
Probably easiest.